src/app/shared/components/drawer/drawer/drawer.component.ts

```diff
@@ -197,6 +197,11 @@ export class DrawerComponent implements AfterViewInit, OnDestroy {
     this.sendToggle();
   }
 
+  resetDrawer() {
+    this.opened = true;
+    this.sendToggle();
```

Comment:
Is this call really needed? It is called implicitly when opened/extended changes.